modules/gitops-registering-an-additional-oauth-client.adoc

```diff
@@ -5,7 +5,7 @@
 [id="registering-an-additional-oauth-client_{context}"]
 = Registering an additional an OAuth client
```

Comment:
This is out of scope so I won't stop the merging over this, but it seems like this heading has a typo:
```suggestion
= Registering an additional OAuth client
```